packages/snaps-execution-environments/src/node-process/index.ts

```diff
@@ -1,8 +1,7 @@
-import { executeLockdown } from '../common/lockdown/lockdown';
 import { executeLockdownMore } from '../common/lockdown/lockdown-more';
 import { ChildProcessSnapExecutor } from './ChildProcessSnapExecutor';
 
-executeLockdown();
+// Lockdown is already applied in LavaMoat
```

Comment:
:thinking: makes me wonder if it has all the configurations set to the same values you needed before.

Reply:
There looks to be a slight difference in configuration: https://github.com/LavaMoat/LavaMoat/blob/5c48a1ecfdc2ebd1a5a8d2d82d1eebdefcf82cb4/packages/core/src/kernelTemplate.js#L47-L58

https://github.com/MetaMask/snaps-monorepo/blob/main/packages/snaps-execution-environments/src/common/lockdown/lockdown.ts#L14-L18

Reply:
overrideTaming, probably not important